src/Tribe/Views/V2/Widgets/Widget_List.php

```diff
@@ -78,28 +85,80 @@ class Widget_List extends Widget_Abstract {
 		'jsonld_enable'        => 'tribe_is_truthy',
 	];
 
-	/**
-	 * @todo update in TEC-3612 & TEC-3613
-	 *
-	 * {@inheritDoc}
-	 */
-	public function setup() {
-
-	}
-
 	/**
 	 * {@inheritDoc}
 	 */
-	public function get_arguments() {
+	public function get_arguments( $instance = [] ) {
 		$arguments = $this->arguments;
 
 		$arguments['description'] = esc_html__( 'A widget that displays upcoming events.', 'the-events-calendar' );
 		// @todo update name once this widget is ready to replace the existing list widget.
 		$arguments['name']                          = esc_html__( 'Events List V2', 'the-events-calendar' );
 		$arguments['widget_options']['description'] = esc_html__( 'A widget that displays upcoming events.', 'the-events-calendar' );
 
+		// Setup default title.
+		$arguments['title']        = __( 'Upcoming Events', 'the-events-calendar' );
+
+		// Setup admin fields.
+		$arguments['admin_fields'] = [
+			'title'                => [
+				'label' => __( 'Title:', 'the-events-calendar' ),
+				'type'  => 'text',
+			],
+			'limit'                => [
+				'label' => __( 'Show:', 'the-events-calendar' ),
+				'type'  => 'dropdown',
```

Comment:
:warning: Array double arrow not aligned correctly; expected 4 space(s) between "'type'" and double arrow, but found 2.

<sub>hash: 9ea762ae1b9f02f7bc4a115a1914954a</sub>